.github/workflows/helm-docs.yaml

```diff
@@ -32,15 +31,101 @@ jobs:
 
           curl --output helm-docs.tar.gz --location https://github.com/norwoodj/helm-docs/releases/download/v1.5.0/helm-docs_1.5.0_Linux_x86_64.tar.gz
           tar -xvf helm-docs.tar.gz
-          # Verify install
+          # Verify installation
           ./helm-docs --version
 
       - name: Generate Helm Docs
         run: |
-          ./helm-docs/helm-docs
+          # Generate README.md based on Chart.yaml and template
+          ./helm-docs/helm-docs --template-files=./.helm-docs/templates.gotmpl --template-files=.helm-docs.gotmpl --template-files=./.helm-docs/README.md.gotmpl
+
+      - name: Generate Demo-Apps Docs
```

Comment:
Would love it if we also included a way to run these locally for anybody who doesn't want to rely on the helm-docs bot. E.g. include a helm-docs target in the make file which runs these steps.

Reply:
Me too :-) but it's not merged by now...

Reply:
Mh ok right, something for the future then